pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -107,6 +113,22 @@ func Convert_kops_CNINetworkingSpec_To_v1alpha1_CNINetworkingSpec(in *kops.CNINe
 	return autoConvert_kops_CNINetworkingSpec_To_v1alpha1_CNINetworkingSpec(in, out, s)
 }
 
+func autoConvert_v1alpha1_CalicoNetworkingSpec_To_kops_CalicoNetworkingSpec(in *CalicoNetworkingSpec, out *kops.CalicoNetworkingSpec, s conversion.Scope) error {
```

Comment:
Not sure exactly what you're asking.  We _should_ have a CI check that the apimachinery autogenerated code is up to date.  If you change the API, you're responsible for writing the conversion code for that field if any is needed (because it's part of the work of adding the field).  When we create a new version of the API (v1alpha3), we copy v1alpha2 and go from there.  So there shouldn't be ongoing maintenance.